scripts/common_test.py

```diff
@@ -1392,3 +1392,17 @@ def test_is_oppia_server_already_running_when_a_port_is_open(
                 lambda port: port == common.GAE_PORT_FOR_E2E_TESTING))
 
             self.assertTrue(common.is_oppia_server_already_running())
+
+    def test_start_subprocess_for_result(self) -> None:
+        process = subprocess.Popen(
+            ['echo', 'test'], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        def mock_popen( # pylint: disable=unused-argument
+            unused_cmd_tokens: List[str], stdout: int, stderr: int
```

Comment:
No need to use parameters of the form `unused_*` when you have the `pylint: disable=unused-argument` comment

Reply:
Done.